ntfs_fr.md

```diff
@@ -422,6 +422,7 @@ physical_mode_id | chaine | Requis | Identifiant du mode physique (lien vers le
 trip_property_id | chaine | Optionnel | Identifiant de la propriété accessibilité (lien vers le fichier [`trip_properties.txt`](#trip_propertiestxt-optionnel))
 dataset_id | chaine | Requis | Identifiant du jeu de données ayant fourni la circulation (lien vers le fichier [`datasets.txt`](#datasetstxt-requis)).
 geometry_id | chaine | Optionnel | Identifiant du tracé représentant la circulation (lien vers le fichier [`geometries.txt`](#geometriestxt-optionnel))
+trip_pattern_id | chaine | Optionnel | Identifiant de la mission
```

Comment:
```suggestion
trip_pattern_id | chaine | Optionnel | Identifiant de la mission (ie. une séquence ordonnée d'arrêt ayant les mêmes propriétés et parfois connue du voyageur)
```

Reply:
what about sticking with Transmodel and `journey_pattern_id` ?

Reply:
Oops, not intended. I completely forgot that. Fixed. I also added the complement of definition of what is a journey pattern.